src/ResourceManager/OperationalInsights/Commands.OperationalInsights/DataSources/NewDataSourceCmdletsPerKind/NewAzureOperationalInsightsAzureActivityLogDataSourceCommand.cs

```diff
@@ -22,6 +22,7 @@ namespace Microsoft.Azure.Commands.OperationalInsights
 {
     [Cmdlet(VerbsCommon.New, "AzureRmOperationalInsightsAzureActivityLogDataSource", SupportsShouldProcess = true, 
         DefaultParameterSetName = ByWorkspaceName), OutputType(typeof(PSDataSource))]
+    [Alias("New-AzureRmOperationalInsightsAzureAuditDataSource")]
```

Comment:
You also need to add the alias in script, use the IModuleAssemblyInitializer pattern, as in the Profile and Resources cmdlets.